config/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:stretch
+FROM python:3.7
```

Comment:
can you try:
FROM camptocamp/geomapfish-tools:2.5.0.77

to be consistent with the rest of the structure ?

Reply:
actually the scaffold system of gmf2.5 places the config image into . instead of ./config and builds from gmf-tools

https://github.com/camptocamp/c2cgeoportal/blob/2.5/geoportal/c2cgeoportal_geoportal/scaffolds/create/build_tmpl#L109
https://github.com/camptocamp/c2cgeoportal/blob/2.5/geoportal/c2cgeoportal_geoportal/scaffolds/create/Dockerfile_tmpl#L1

Reply:
There is no "camptocamp/geomapfish-tool:2.5". I have to pick a very specific tag like 2.5.0.80. Also it weight 2.25Go :scream: